Jenkinsfile

```diff
@@ -2,97 +2,141 @@
 def utils = new io.fabric8.Utils()
 def flow = new io.fabric8.Fabric8Commands()
 def project = 'fabric8-ui/fabric8-planner'
-def ciDeploy = false
 def tempVersion
```

Comment:
This could be removed. `tempVersion` is already declared on https://github.com/fabric8-ui/fabric8-planner/pull/2514/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R46